src/server/game/Grids/Notifiers/GridNotifiers.cpp

```diff
@@ -130,6 +130,10 @@ inline void CreatureUnitRelocationWorker(Creature* c, Unit* u)
     if (c->HasReactState(REACT_AGGRESSIVE) && !c->HasUnitState(UNIT_STATE_SIGHTLESS))
         if (c->IsAIEnabled && c->CanSeeOrDetect(u, false, true))
             c->AI()->MoveInLineOfSight_Safe(u);
+        else
```

Comment:
GridNotifiers.cpp:133:9: fatal error: 
      add explicit braces to avoid dangling else [-Wdangling-else]
        else